Apps/Src/SendTritium.c

```diff
@@ -232,105 +233,134 @@ static void readInputs()
 {
 
     // Update pedals
-    static uint32_t brakeSaturationCt = 0;
+    static uint8_t brakeSaturationCt = 0;
+    static uint8_t accelSaturationCt = 0;
 
-    uint32_t latest_pedal = Pedals_Read(BRAKE);
+    uint8_t latest_pedal = Pedals_Read(BRAKE);
 
     if (brakeSaturationCt < 3)
     {
-        if (latest_pedal == 100)
-            brakeSaturationCt++;
+        if (latest_pedal == 100) 
+            {brakeSaturationCt++;}
```

Comment:
Not sure what our current style guidelines are, but I think this if statement structure is different than what is in the rest of the code base.